.github/workflows/full-downstream.yml

```diff
@@ -35,23 +35,23 @@ jobs:
     name: Full downstream build
     steps:
       - name: Clean Disk Space
-        uses: kiegroup/kogito-pipelines/.ci/actions/ubuntu-disk-space@1.13.x
+        uses: kiegroup/kie-ci/.ci/actions/ubuntu-disk-space@main
         if: ${{ matrix.os == 'ubuntu-latest' }}
       - name: Support long paths
         if: ${{ matrix.os == 'windows-latest' }}
-        uses: kiegroup/kogito-pipelines/.ci/actions/long-paths@1.13.x
+        uses: kiegroup/kie-ci/.ci/actions/long-paths@main
       - name: Java and Maven Setup
-        uses: kiegroup/kogito-pipelines/.ci/actions/maven@1.13.x
+        uses: kiegroup/kie-ci/.ci/actions/maven@main
         with:
           java-version: ${{ matrix.java-version }}
           maven-version: ${{ matrix.maven-version }}
           cache-key-prefix: ${{ runner.os }}-${{ matrix.java-version }}-maven${{ matrix.maven-version }}
       - name: Build Chain
-        uses: kiegroup/kogito-pipelines/.ci/actions/build-chain@1.13.x
+        uses: kiegroup/kie-ci/.ci/actions/build-chain@main
         with:
```

Comment:
Just noticed, I think we miss `definition-file` here

Reply:
Added `run_fdb` label, it should fail with "missing definition file" or something similar